cmd/command.go

```diff
@@ -6,8 +6,13 @@ import (
 	"os"
 )
 
+// PreInit defines all the flags that are going to be used.
+// If the default value is not going to be used,
+// the flags will be set as part of the flags.Parse function.
+// In Args, several paths will be changed to be under the updated layers directory (if the user didn't set them using a flag)
```

Comment:
Do you think the update should be part of the Args function, or it's better to add a new function to the interface and make the update over there?
And if we should keep the update as part of the Args function, maybe its name should be changed?